quickstart.py

```diff
@@ -93,7 +93,7 @@ def is_successful(row, keys):
     try:
         successful = row[i] != 'Not yet' and row[j] == 'Yes'
     except IndexError:
-        successful = false
+        successful = False
```

Comment:
I have no idea how I missed this. Anyhow, this setting to `False` is just for sanity checking the successful columns on the Google Sheet.